projects/demo/project.json

```diff
@@ -287,6 +195,16 @@
                 "collectCoverage": false,
                 "testMatch": ["<rootDir>/{projectRoot}/src/**/*.spec.ts"]
             }
+        },
+        "serve-static": {
```

Comment:
Do we use it anywhere?

What is the advantage compared to using an [already existing server](https://github.com/taiga-family/ci/blob/main/actions/run/serve/action.yml#L43):

```
$ npx -y local-web-server --port 4200 --directory dist/demo/browser --spa index.html
```

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/12411/commits/5f4b12a89018bbb3c85473e1d78666ddd65564e3